docs/support/troubleshooting.md

```diff
@@ -160,3 +160,16 @@ net.core.rmem_default=8388608 // only add if CycloneDDS is confgured
     ```bash
     echo $RMW_IMPLEMENTATION  // if Cyclone DDS is configured, this command will return "rmw_cyclonedds_cpp"
     ```
+
+### Multicast is disabled
+
+If you get the error message `selected interface "lo" is not multicast-capable: disabling multicast`, run the following command to allow multicast.
+
+```bash
+sudo ip link set multicast on lo
+```
+
+!!! note
+
+    You might have to change the interface name. In this case, it's `lo`.
+    Adjust the interface name based on your environment and the error message.
```

Comment:
As per comments above, if the more generalized error message and command are acceptable, then this note can be removed entirely.